src/sql/workbench/contrib/query/browser/queryEditor.ts

```diff
@@ -563,7 +563,10 @@ export class QueryEditor extends EditorPane {
 
 	// helper functions
 
-	public isSelectionEmpty(): boolean {
+	/**
+	 * Returns a boolean value indicating whether the editor is empty.
+	 */
+	public isEditorEmpty(): boolean {
```

Comment:
the previous function name doesn't accurately represent its actual job, so fixing it to avoid confusion.